python/requirements/ml/dl-gpu-requirements.txt

```diff
@@ -14,3 +14,6 @@ torch-scatter==2.1.1+pt20cu118
 torch-sparse==0.6.17+pt20cu118
 torch-cluster==1.6.1+pt20cu118
 torch-spline-conv==1.2.2+pt20cu118
+
+# Install vLLM for documentation example tests (doc/source/serve/doc_code).
+vllm==0.4.2 
```

Comment:
the serve build doesn't install this requirement though so the test will likely fail; you might need to add here https://github.com/ray-project/ray/blob/master/ci/docker/serve.build.Dockerfile#L26

-r python/requirements/dl-cpu-requirements.txt

and add `python/requirements/dl-cpu-requirements.txt` to the source file in https://github.com/ray-project/ray/blob/master/ci/docker/serve.build.py39.wanda.yaml#L13


also perhaps add `vllm==0.4.2` to dl-cpu-requirements.txt because dl-gpu-requirements.txt doesn't install on a cpu image (the serve.build.Dockerfile is based on ubuntu not cuda)

Reply:
sorry about all of these complexities

Reply:
we only need it for the GPU build, do I still need to add it?

Reply:
ah yess, your docs run on a gpu build, ignore what i say :D

Reply:
looks like the build failed w/ the new compiled requirements: https://buildkite.com/ray-project/premerge/builds/26291#018f879a-16e5-4a7b-b63a-06f2e79a2c9a/119-1330

Reply:
it requires the latest version of torch as well, this vllm package is pretty picky ;)

Reply:
yeah :(